data/mods/MindOverMatter/powers/telepathy.json

```diff
@@ -377,7 +377,8 @@
     "energy_increment": -250,
     "base_casting_time": 120,
     "final_casting_time": 75,
-    "casting_time_increment": -3.5
+    "casting_time_increment": -3.5,
+    "ignored_monster_species": [ "ZOMBIE", "ROBOT", "NETHER", "NETHER_EMANATION", "LEECH_PLANT", "WORM", "FUNGUS", "SLIME" ]
```

Comment:
`NETHER` monsters often have minds.  Though I guess it might be accurate to say that their intelligence might be too alien to our minds for telepathic damage to impact them.

Reply:
The latter is what I was going for--that Nether monsters could be telepathically attacked (unsubtle, brute-force, swinging a two-by-four) but not telepathically mind-controlled (subtle, delicate, wielding a scalpel)